sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample00_AuthenticateClient.md

```diff
@@ -0,0 +1,56 @@
+# Authenticate the client
```

Comment:
Sample 00, eh?   That's cheating!    We may want to consider naming this "Hello World" to snap closer to the [samples guidelines](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-samples) - though I don't feel strongly about it, personally.

Reply:
Ah I didn't realize that was an official guideline but I have to say I'm not a big fan of it.

Reply:
No objections if we choose to ignore it here.